packages/react-aria-components/src/Calendar.tsx

```diff
@@ -21,15 +21,19 @@ import {HeadingContext} from './Heading';
 import React from 'react';
 import {TextContext} from './Text';
 
-export interface CalendarProps<T extends DateValue> extends Omit<BaseCalendarProps<T>, 'errorMessage'>, RenderProps<CalendarState>, SlotProps {
+export interface CalendarRenderProps extends Omit<CalendarState, 'setValue' | 'setFocusedDate' | 'focusNextDay' | 'focusPreviousDay' | 'focusNextRow' | 'focusPreviousRow' | 'focusNextPage' | 'focusPreviousPage' | 'focusSectionStart' | 'focusSectionEnd' | 'focusNextSection' | 'focusPreviousSection' | 'selectFocusedDate' | 'selectDate' | 'setFocused'> {}
```

Comment:
instead of all these omits, which if we ever add to the calendar state something we don't want RAC renderProps to pick up, we'd need to add here, we could instead create a more base type that CalendarState extends

naming is the hard part, haha...

Reply:
I don't follow the "but ..." it sounds like the same thing we just agreed on?

Reply:
I wasn't following the part about creating a base type that CalendarState extends